include/GameplayState.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "BaseGameState.h"
+
+// This is where the main gameplay will occur
+class GameplayState : public GameState
+{
+public:
+	// Could pass in state such as settings that were
+	// toggled on the start screen or stuff like that
+	// Make un-default when actually needed
+	GameplayState() = default;
+
+	void handleEvent(SDL_Event& event) override;
```

Comment:
This should probably be const, unless we want to be able to manipulate the event data.